### PR TITLE
254 migration wrapup

### PR DESCRIPTION
**Pull request by slifty**
This is a pretty short and sweet PR -- removes the empty seed.sql and provides some documentation around seed data.

### CODE REVIEW

**Comment by jasonaowen** [comment]
We fixed the issue causing the failing unit tests in #929; you'll need to rebase onto master to pull that fix in.

File: INSTALL.md [hunk 1]
```diff
@@ -372,8 +372,7 @@ EOF
 1. Create database schema and initial data.  Right now, we are midway
    through a migration to Liquibase and so are using both Liquibase
    migrations and SQL files to create the database. First, run the
-   migrations to create the first set of tables.  Then, use `seed.sql`
-   to create tables and data for the application, and `jbpm.sql` to
+   migrations to create the first set of tables.  Then, use  `jbpm.sql` to
```
Comment (jasonaowen) [inline INSTALL.md]: I think the "first set of tables" language was used to describe the transition state. Now that we're fully transitioned, I think we can adopt the language we were previously using for `seed.sql`: "Run the migrations to create the tables and data for the application."

Relatedly, I think there isn't an order dependency between Liquibase migrations and `jbpm.sql`; they don't overlap or interact at all, so they can be run in any order. I don't know if that's worth calling out explicitly, but maybe we could soften the language around "first do this, then do that" to something "do this and that"?

File: docs/MIGRATIONS.md [hunk 2]
```diff
@@ -42,6 +42,14 @@ The reasoning for this [is explained on this liquibase blog post](http://www.liq
 The goal of a Liquibase change log file is to track the linear sequence of changes required to take a database from a starting point to the current state, and it is built up one change at a time throughtout development. By following the “always append changes” rule you will ensure that all databases are consistent since all will have gone through the exact same set of changes.
 ```
 
+## Adding Seed Data
+
+Liquibase makes it easy to insert seed data into tables. There is a separate migration file for seed data: `db/changelog/db-changelog-seed.xml`. Seed data itself is stored in the `db/seeds/` directory through csvs.  The first row of a seed data CSV should contain the field names with corresponding data.
+
+The changeset type for seed data is [`loadUpdateData`](https://www.liquibase.org/documentation/changes/load_update_data.html) which makes it easy to add or modify seed data to reflect the contents of the seed CSV.
+
+Liquibase is smart enough to parse certain key words, e.g. a value of "NULL" will insert null for that column.
```
Comment (jasonaowen) [inline docs/MIGRATIONS.md]: Thank you for documenting all this!

**Comment by jasonaowen** [comment]
@slifty I think we also need to update `psm-app/db/README.md` as part of removing `seed.sql`:

> ## Seed.sql
> This file sets up the database, inserts static data, and create some default users to get you going

File: INSTALL.md (hunk 1 above)
Comment (slifty) [reply INSTALL.md]: Updated the language -- I do think the core tables and the jbpm tables are separate "sets" of tables, but I've clarified.  Also added in the mention that order doesn't matter.

**Pull request merged**